CloverSDK/Utils/CLVServerEnvironment.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  ServerEnvironment.swift
+//  CloverDashboard
+//
+//  Created by Yusuf on 12/17/15.
+//  Copyright © 2015 Clover Network, Inc. All rights reserved.
+//
+
+public enum CLVServerEnvironment: String {
+  case US = "https://www.clover.com"
+  case EU = "https://www.eu.clover.com"
+  case DEV1 = "https://dev1.dev.clover.com"
+  case STG1 = "https://stg1.dev.clover.com"
```

Comment:
I was expecting some discussion about this. I also thought about that, but couldn't find a good solution. Unfortunately, init() wouldn't work, because in the current swift version, enum return type has to be statically defined. You can have an enum which takes arguments, but then it may not have a return value, which means we would have to pass in all url strings, not just dev1 and stg1, which defeats the purpose. This was actually discussed in swift forums, I'm hoping they might implement that in the future versions.

In our case, if using them in a public place is a concern, then I will remove the dev1 and stg1 from here.